### ISSUE TITLE
Readout Protection

### ISSUE THREAD

**Issue by WGR7** [brother-yan/LGTISP#1]
I've used a translator and as I could see your programmer breaker the code protection for lgt8f, is it right? If this is true, for me this is a problem, I started to prototype with this uC because but has so many good features.

**Comment by brother-yan** [comment]
> I've used a translator and as I could see your programmer breaker the code protection for lgt8f, is it right? If this is true, for me this is a problem, I started to prototype with this uC because but has so many good features.

Yes. My code can read flash(address : 0x0400 ~ 0x7FFF) after power-down. You can use my code to confirm it.

**Comment by WGR7** [comment]
Só, If my code fits in 0-400h will it be safe? How the readout protection works in LGT8F uC family? As I could see, there's no lock bits and the datasheet have no information about it. 

Regards

**Comment by brother-yan** [comment]
> Só, If my code fits in 0-400h will it be safe? How the readout protection works in LGT8F uC family? As I could see, there's no lock bits and the datasheet have no information about it.
> 
> Regards

I'm not sure. I don't known how LGT8Fx8P works. The official share just very little information.
I have no time to crack 0x0000~0x0400 flash recently.

**Comment by WGR7** [comment]
Can your tool get the data at configured eeprom space?
